application/backup.sh

```diff
@@ -1,4 +1,29 @@
 #!/usr/bin/env sh
+# Error Tracking
+if [ ! -z "${ SENTRY_DSN }"]; then
+    eval "$(curl -sL https://sentry.io/get-curl.sh)"
+    SENTRY_TRACE_ID=$(uuidgen)
+fi
+
+# Send error to Sentry
+error_to_sentry() {
+    local error_message="$1"
+    local db_name="$2"
+    local status_code="$3"
+
+    if [ ! -z "${ SENTRY_DSN }" ]; then
+        curl -X POST "${ SENTRY_DSN }" \
```

Comment:
Spaces that shouldn't be there.
```suggestion
    if [ ! -z "${SENTRY_DSN}" ]; then
        curl -X POST "${SENTRY_DSN}" \
```